gui/mozregui/ui/global_prefs.ui

```diff
@@ -69,6 +69,23 @@
        </property>
       </widget>
      </item>
+     <item row="3" column="0">
+      <widget class="QLabel" name="label_2">
+       <property name="toolTip">
+        <string>If checked, background downloads of the next builds will start while you are evaluating a build.</string>
+       </property>
+       <property name="text">
+        <string>background downloads:</string>
+       </property>
+      </widget>
+     </item>
+     <item row="3" column="1">
+      <widget class="QCheckBox" name="bg_downloads">
+       <property name="text">
+        <string/>
+       </property>
+      </widget>
+     </item>
```

Comment:
hm, maybe not, you're right. But since we have the option in the CLI, I guess it make sense to have it too.

Maybe we could do an 'advanced options' panel that can hide this option as a default ? And clicking on 'advanced options' would show it. I think other options can be in there, like the http timeout.